runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -156,6 +159,25 @@ public void onWidgetLoaded(WidgetLoaded event)
 		}
 	}
 
+	@Subscribe
+	public void onGameTick(GameTick tick)
+	{
+		if (config.callChangeAudioCue() && currentRound != null)
+		{
+			long timeToChange = currentRound.getTimeToChange();
+
+			if (timeToChange == 30 && !playingCue)
```

Comment:
Since `30` is a constant that should be shared between this class and Round, make it a package-private constant and reference it throughout both classes.